go/pserver/client/c/test/test_train.py

```diff
@@ -31,6 +31,13 @@ def main():
 
     # create optimizer of new remote updater to pserver
     optimizer = paddle.optimizer.Momentum(momentum=0, learning_rate=1e-3)
+    optimizer.set_remote_optimizer_config(
```

Comment:
In [OptimizerConfig](https://github.com/PaddlePaddle/Paddle/blob/develop/proto/OptimizerConfig.proto#L125), each "parameter" object have a single configuration details, each "parameter" can have different algorithm like "SGD" or "Adagrad".

In V1 [TrainerConfig.proto](https://github.com/PaddlePaddle/Paddle/blob/develop/proto/TrainerConfig.proto#L23) configs the global optimization algorithm for all parameters, and in [ParameterConfig.proto](https://github.com/PaddlePaddle/Paddle/blob/develop/proto/ParameterConfig.proto#L37) set config for each "parameter", but only `learning_rate`, `decay_rate` and `momentum` values.